config/locales/errors.yml

```
@@ -8,6 +8,11 @@ en:
       If you typed the web address, check it is correct.
       If you pasted the web address, check you copied the entire address.
     title: Page not found
+  repeat_submission:
+    body: We deleted your answers because they've been submitted successfully already.
```

Not essential (if you ask me), but would be nice to make this straight apostrophe a curly one. ie: they’ve

---

How do you type that!

---

`Alt + ]` -> `‘`
`Alt + Shift + ]` -> `’`
`Alt + [` -> `“`
`Alt + Shift + {` -> `”` 

```suggestion
    body: We deleted your answers because they’ve been submitted successfully already.
```

---

(or do what I do and copy it from someplace...! ) Thanks both!